jib-core/src/main/java/com/google/cloud/tools/jib/registry/RegistryEndpointCaller.java

```diff
@@ -102,10 +138,18 @@ T call() throws IOException, RegistryException {
     return call(initialRequestState);
   }
 
-  /** Calls the registry endpoint with a certain {@link RequestState}. */
+  /**
+   * Calls the registry endpoint with a certain {@link RequestState}.
+   *
+   * @param requestState the state of the request - determines how to make the request and how to
+   *     process the response
+   * @return an object representing the response, or {@code null}
+   * @throws IOException for most I/O exceptions when making the request
```

Comment:
should these be @link?

Reply:
do you mean like `@throws IOException for most {@link IOException}s when making the request`?

Reply:
nope. Please ignore. I shouldn't make review comments so early in the morning.